src/apps/js_generic/js_generic.cpp

```diff
@@ -969,6 +969,7 @@ namespace ccfapp
       JSAutoFreeRuntime auto_free_rt(rt);
 
       JS_SetMaxStackSize(rt, 1024 * 1024);
+      JS_SetMemoryLimit(rt, 20 * 1024 * 1024);
```

Comment:
This seems low, we typically run with 500Mb of heap, can we make it 100Mb at least?